lib/ldp.js

```diff
@@ -266,11 +266,15 @@ class LDP {
     var root = !ldp.idp ? ldp.root : ldp.root + host + '/'
     var filePath = utils.uriToFilename(resourcePath, root, host)
 
-    // PUT requests not supported on containers. Use POST instead
-    if (filePath.endsWith('/')) {
-      return callback(error(409,
-        'PUT not supported on containers, use POST instead'))
+    const statusCode = fs.existsSync(filePath) ? 204 : 201
```

Comment:
Yes, indeed… I don't mind either way. Can't wait to switch to Node 8 though, where sync and async have essentially the same syntax 😄